test/src/materials/TestNEMLStressSimple.C

```diff
@@ -0,0 +1,52 @@
+/****************************************************************/
```

Comment:
Do you need this class anymore?  Seems like everything this does (and then some) is in NEMLStress now.

Reply:
we decided to keep it as an example